src/app/core/ui/search/search.component.html

```diff
@@ -1,26 +1,89 @@
-<div fxHide.xs="true" fxHide="false">
-  <mat-form-field fxFill class="search-field">
-    <span matPrefix><mat-icon class="search-icon" fontIcon="fa-search"></mat-icon>&nbsp;</span>
-    <input matInput (input)="search()" title="Search" [matAutocomplete]="autoResults" [(ngModel)]="searchText">
+<div class="desktop-search-bar" fxHide.xs="true" fxHide="false">
+  <mat-form-field fxFill class="search-field" floatLabel="never">
+    <span matPrefix>
+      <mat-icon matPrefix class="search-icon" fontIcon="fa-search"></mat-icon>
+    </span>
+    <mat-label>Search</mat-label>
+    <input
+      matInput
+      title="Search"
+      [matAutocomplete]="autoResults"
+      [(ngModel)]="searchText"
+      (ngModelChange)="searchInputChanged($event)"
+      #searchInput
+    />
   </mat-form-field>
 
-  <mat-autocomplete #autoResults="matAutocomplete" (optionSelected)="clickOption($event.option)">
-    <mat-option *ngFor="let res of results" [value]="''">
-      <app-child-block *ngIf="res.getType()==='Child'" [entity]="res" [ngClass]="{'result-inactive': !res.isActive()}"></app-child-block>
-      <app-school-block *ngIf="res.getType()==='School'" [entity]="res"></app-school-block>
+  <mat-autocomplete
+    #autoResults="matAutocomplete"
+    (optionSelected)="clickOption($event.option)"
+  >
+    <mat-option
+      *ngIf="
+        searchInput.value.length > 0 &&
+        searchInput.value.length < MIN_CHARACTERS_FOR_SEARCH
+      "
+    >
+      <p>Insert at least {{ MIN_CHARACTERS_FOR_SEARCH }} characters</p>
```

Comment:
I would give the "operational" mat-options a different styling than the actual search result items.

maybe something like
```
result-hint {
    background-color: lightgrey;
    font-style: italic;
}
```

Reply:
good point, updated the styles - I would stay with the hints